core/2d/CCCamera.cpp

```diff
@@ -42,7 +42,7 @@ Viewport Camera::_defaultViewport;
 Camera* Camera::create()
 {
     Camera* camera = new Camera();
-    camera->initDefault();
+    camera->initCamera();
```

Comment:
Can be still name `initDedault`?

Reply:
it's gonna be misleading if `initDefault` is used, previous code bases will think it's an old API function and will lead to bugs.

Reply:
Actually, it needs to be `init` only overriding the node's `init` function, that way the code is cleaner